.gitmodules

```diff
@@ -965,6 +965,9 @@
 [submodule "vendor/grammars/vscode-lean"]
 	path = vendor/grammars/vscode-lean
 	url = https://github.com/leanprover/vscode-lean
+[submodule "vendor/grammars/vscode-prisma"]
+	path = vendor/grammars/vscode-prisma
+	url = https://github.com/prisma/vscode-prisma
```

Comment:
This project is missing a license. We can't use it without a license (it is currently not open source).

Reply:
Hey @pchaigno, appreciate the help! 

Just asked the guys at prisma to go ahead and add the missing license to the grammar repo. Should be up soon! I'll push an update as soon as they have that finished!